src/main/java/htsjdk/samtools/CRAMBAIIndexer.java

```diff
@@ -253,28 +340,16 @@ private int computeIndexingBin(final Slice slice) {
             return GenomicIndexUtil.regionToBin(alignmentStart, alignmentEnd);
         }
 
-
         /**
          * Record any index information for a given CRAM slice
          *
-         * @param slice CRAM slice, single ref or unmapped only.
+         * Reads these Slice fields:
+         * sequenceId, alignmentStart, alignmentSpan, containerOffset, index
+         *
+         * @param slice CRAM slice, single ref only.
          */
         private void processSingleReferenceSlice(final Slice slice) {
-
-            // metadata
-            indexStats.recordMetaData(slice);

```

Comment:
moved out of this method: `processSingleReferenceSlice()` is not for unmapped slices but we do want to call `recordMetaData()` for those